agent/provision_agent.go

```diff
@@ -42,7 +42,7 @@ func (a *provisionAgent) Run() {
 }
 
 func (a *provisionAgent) Process() bool {
-	scs, err := a.pfclient.FetchContainersFromServer(a.nodeHostname)
+	scs, err := a.pfclient.FetchContainersFromServer(a.nodeHostname, "ListScheduledContainers")
```

Comment:
Hmm.. now I think we should remove if else logic in this method
it's bad to sent `ListScheduledContainers` as an argument

Reply:
Done.